CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Version 2.0-dev
 ===============
 
-The library has been tested using Agda 2.6.2.
+The library has been tested using Agda 2.6.2.2.
```

Comment:
Should be `2.6.3` no?

Reply:
Yes, that is the case now... some of these PRs haven't kept up with the transition to v2.6.3